valohai_yaml/objs/step.py

```diff
@@ -105,6 +106,14 @@ def get_parameter_defaults(self, include_flags: bool = True) -> Dict[str, Any]:
             if parameter.default is not None and (include_flags or parameter.type != 'flag')
         }
 
+    def get_input_defaults(self) -> Dict[str, Any]:
+        """Get a dict mapping parameter names to their defaults (if set)."""
```

Comment:
Copy-pasted docstring? :)

Reply:
🤦🏻‍♂️ I won't be ever copypasting again...